README.md

```diff
@@ -179,9 +179,14 @@ To run locally:
 # Optional: ensure the elasticsearch index is clean
 docker-compose up --build -d elasticsearch
 curl -XDELETE localhost:9200/_all
-# Start the rest of the services
+# Run the 1000 genomes dataset tests for general functionality
```

Comment:
For simplicity, revert changes to this file. I wanted tests for Framingham so the functionality doesn't break, but we don't need to spell out how to run it in the README.